### ISSUE TITLE
Routing for creating and adopting cards

### ISSUE THREAD

**Issue by dnaidionov** [cardstack/cardstack#1283]
- Click on New Blank Card takes user to new card in Schema mode. 
- Adopting a card (e.g. clicking on template or selecting “adopt” action) takes user to new card in Edit mode.

**Comment by dnaidionov** [comment]
Blocked by #1279. This ticket should be picked only after  1279 is done.
